module/Company/view/company/company/list.phtml

```diff
@@ -12,16 +12,36 @@ $this->headTitle($this->translate('Companies'));
 <div class="container">
     <h1><?= $this->translate('Companies') ?></h1>
 
-    <div class="company-grid row">
-        <?php
-
-        foreach ($companyList as $company) {
-            echo $this->partial('partial/company/company/grid/company.phtml',
-                ['company' => $company, 'locale' => $locale]);
-        }
-
-        ?>
-
+    <div class="row">
+        <div class="col-md-3 hidden-sm">
+            <div class="list-group">
+                <?php
+                function sortCompanies(\Company\Model\Company $a , \Company\Model\Company $b) {
+                    return strcasecmp($a->getName(), $b->getName());
+                }
+                $sortedList = $companyList;
+                usort($sortedList, "sortCompanies");
```

Comment:
Why not just modify the database query so the companies are already sorted? E.g. `->addOrderBy('name', 'ASC')`